debug-web-ui/src/components/LookupAccountForm.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react'
+
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import { Table } from 'react-bootstrap';
+
+import ErrorForm from './ErrorForm.js';
+import SearchField from './SearchField.js';
+
+
+const loadAccount = (id, api) => {
+    return api.lookupAccount(id)
+}
+
+const LookupAccountForm = ({api}) => {
+    const [state, setState] = useState({account: undefined, error: undefined, loading: false});
+
+    const lookupSuccess = (response) => setState({account: response.data, error: undefined});
+    const lookupFail = (error) => setState({account: undefined, error: error})
+
+    return (
+        <div>
+            <SearchField placeholder="AccountID"
+                         onClick={(id) => loadAccount(id, api)
+                                            .then(lookupSuccess)
+                                            .catch(lookupFail)} />
+            <hr />
+            {state.account && <DetailsForm account={state.account} />}
+            {state.error && <ErrorForm error={state.error} />}
+        </div>
```

Comment:
There is a way to make try/catch pattern in jsx: https://reactjs.org/docs/error-boundaries.html
But I like your way much more - because it's more Go-style - handle error where you faced it.

Reply:
I changed it a bit, so we can have a catch-all code, so now we kind of have both. But if you think that the previous approach was better, I can change back.